shapes/go-cam-shapes.shex

```diff
@@ -434,7 +435,8 @@ PREFIX results_in_remodeling_of: <http://purl.obolibrary.org/obo/RO_0002591>
 }
 <Protein> @<ChemicalEntity> AND EXTRA a {
```

Comment:
We might just do away with Protein unless there is a use for it that is different from information biomacromolecule.